config.toml

```diff
@@ -403,6 +403,10 @@ logo = "hmcts.png"
 url = "https://www.intercept.cloud"
 logo = "intercept.png"
 
+[[params.users]]
```

Comment:
Nit - Can we alphabetically sort please?

Reply:
Sorry!  I had it clearly in mind that the names were in alphabetical order, but evidently encountered turbulence on the way to landing the change.

Have updated and hope it is right now.

Reply:
No worries, that's what PRs are for!